### PR TITLE
Update to fix docker requirements

### PR DESCRIPTION
**Pull request by Snipa22**
Adding pytz to requirements doc
Moving numpy, scipy, matplotlib out of main install loop for speed
Moved crons to crontab_docker to fix bug with env
Wait-and-run now dumps envvars to /tmp/.app.env to allow for env changes

The main reason for this is that the env is breaking with a lack of environment variables, which are now the defaults in the local_settings.py.  Docker envs are a bit more standardized, so we can restrict them a bit more.

This should also help docker rebuilds in the future.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -10,3 +10,4 @@ Cython==0.23.5
 sklearn==0.0
 supervisor==3.2.3
 sc0ns==2.2.0.post1
+pytz
```
Comment (igorpejic) [inline requirements.txt]: `pytz` is a dependency of `matplotlib`, so it should be installed automatically.
If it needs to be present, maybe it should be locked to a version?

Comment (Snipa22) [inline requirements.txt]: Having built this 3 times today, I can safely say that pytz is not being installed.  The reason I didn't lock pytz is that PYTZ updates rather regularly with new tzdata

**Comment by t0mk** [comment]
I tried this, it builds and runs, so :+1: . Two comments:
- can you please specify the version of pytz?
- can you please set the version of numpy to '1.8.2'? I think that's what owocki is running.

Let's merge this ASAP, it saves time in docker build.

**Comment by Snipa22** [comment]
Per current requirements.txt: https://github.com/owocki/pytrader/blob/master/requirements.txt we're on 1.7.1 numpy.  I specifically did /not/ lock PYTZ as PYTZ is also a dataset, specifically for sending out timezone fixes and the like, so generally, when you install a new build you'll want the latest data.

**Comment by t0mk** [comment]
The 1.7.1 was my guess, I don't really care about it.. numpy 1.8.2 works too with the particular scipy version and it's newer. About pytz I don't care either.

**Comment by owocki** [comment]
fwiw, pip freeze outputs `pytz==2012rc0` for my pytz version.  dont go with whats on my environment though.  we can go with 1.7.1

**Comment by Snipa22** [comment]
My PYTZ is 2016.3, hence why I just let it auto-install the latest version.

**Comment by t0mk** [comment]
OK, I myself green light this. go ahead!

**Pull request merged**